woof-code/rootfs-petbuilds/conky-gtk_lua/petbuild

```diff
@@ -0,0 +1,11 @@
+download() {
+    # leave petbuild download empty since we already have the whole source
+    echo -n
+}
+
+build() {
+    # enable conky autostart
+    # rest of the setup is handled by conky-gtk on its first run
+    mkdir -p /root/Startup/
+    cp ./root/.config/conky-gtk/startup_script_format /root/Startup/conkystart
```

Comment:
You can move this file to woof-code/rootfs-petbuilds/conky-gtk_lua/root/Startup/conkystart, then change the petbuild to

```
download() {
     :
}

build() {
     :
}
```

Reply:
The default location of that file should be `root/.config/conky-gtk/startup_script_format`, and its copy/symlink should be made to `root/Startup/conkystart`.

This is due to how the "Enable Autostart" and "Disable Autostart" features are handled in `usr/bin/conky-gtk`:

https://github.com/puppylinux-woof-CE/woof-CE/pull/4239/files#diff-a9e48948c1d2baed4b52d9e3659c261ab1993d03965dc63f88e6c0f57f5adf5eR99-R101

https://github.com/puppylinux-woof-CE/woof-CE/pull/4239/files#diff-a9e48948c1d2baed4b52d9e3659c261ab1993d03965dc63f88e6c0f57f5adf5eR255

Reply:
So you can `ln -s` to it when enabled, remove the symlink when disabled, and keep a symlink in the petbuild instead of creating it within build()

Reply:
My main intention to avoid symlink and use hard link was taking in consideration that in case the user destroys the file, this will be a backup for them.

Please take this fact into consideration and then suggest if I should use symlink.

Reply:
Nobody likes hard links, they can be super confusing. And symlink vs. copying - I don't think this matters too much because you can't prevent users from editing the script anyway. You can't protect users from themselves - if they tinker in things they don't understand, they need to accept the fate of problems.